### ISSUE TITLE
Add `py.typed` to expose typing information to clients

### ISSUE THREAD

**Issue by kkirsche** [statelyai/xstate-python#56]
Per [PEP 561](https://peps.python.org/pep-0561/) packages which distribute type information must have a `py.typed` file which is included in the distributed package. This is specifically discussed in the [Packaging Type Information](https://peps.python.org/pep-0561/#packaging-type-information) section if you'd like to review this in more detail.

This issue is to ask if you are open to this before submitting a merge request to add this so that type information is exposed to user's projects

Thanks!

**Comment by davidkpiano** [comment]
Yes I am, but we will be updating this package significantly, so it may be better to wait until then.

**Comment by leetrout** [comment]
@davidkpiano I know it's everyone's favorite question.... but do you know when that work might be happening? ☺️
